Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.19 as build
+
+ENV CGO_ENABLED=0
+WORKDIR /usr/src/app
+
+COPY go.mod go.sum .
+RUN go mod download && go mod verify
+
+COPY ./ .
+RUN go build -v -o /usr/local/bin/exercism ./exercism
+
+CMD ["/usr/local/bin/exercism"]
```

Comment:
I'm not sure this is correct. I was expecting to be able to `docker run -v ~/exercism:/root exercism <SUBCOMMAND> <OPTIONS>` where `exercism` is the docker image name. Instead I had to say both the image name and the binary name.